doc/index.rst

```diff
@@ -95,6 +95,21 @@ RSS
 **rss/channel/generator**
     A string indicating the program used to generate the channel. (e.g. MightyInHouse Content System v2.3).
 
+**rss/channel/language**
+    Podcast language.
+
+**rss/channel/itunes:author**
+    The group responsible for creating the show.
+
+**rss/channel/itunes:owner**
+    The podcast owner contact information.
```

Comment:
The [spec](http://web.archive.org/web/20120323002653if_/http://www.apple.com/itunes/podcasts/specs.html#owner) says: "This tag contains information that will be used to contact the owner of the podcast for communication specifically about their podcast. It will not be publicly displayed."

Reply:
Changed to ```The podcast owner contact information.
    The <itunes:owner> tag information is for administrative communication about the podcast and isn't displayed in Apple Podcasts```
    According to [the spec](https://help.apple.com/itc/podcasts_connect/#/itcb54353390)